### DISCUSSION TITLE
Help with an urgent Usage Limit Warning

### DISCUSSION THREAD

**Discussion by lambo81** [vercel/community#708] [Help] [answered]
I received an email stating "Edge Middleware invocations cannot use more than 50ms of CPU time on average"

I followed the link to the dashboard but didn't see anything related to the above.  Is this talking about my _middleware.ts file?

I really don't know what could be taking so long in that function.  Could this be avoided with a paid account?

**Comment by amyegan** [comment, marked as answer]
It seems like that email was sent in error and you're not the only one seeing it. The team is currently investigating and I'll update here when I have more information.
